### PR TITLE
Aws documentation

### PR DESCRIPTION
**Pull request by pascalwilbrink**

### CODE REVIEW

File: AWS.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+"executionRoleArn": "arn:aws:iam::[PROFILE_ID]:role/ecsTaskExecutionRole",
+```
+
+Change the executionRoleArn in the following files:
+
+* [aws/task-executions/compas-auto-alignment.json](./aws/task-executions/compas-auto-alignment.json)
+* [aws/task-executions/compas-cim-mapping.json](./aws/task-executions/compas-cim-mapping.json)
+* [aws/task-executions/compas-openscd.json](./aws/task-executions/compas-openscd.json)
+* [aws/task-executions/compas-scl-data-service.json](./aws/task-executions/compas-scl-data-service.json)
+* [aws/task-executions/compas-scl-validator.json](./aws/task-executions/compas-scl-validator.json)
```
Comment (dlabordus) [inline AWS.md:36]: Maybe add some instructions how to import these JSON Files using the console.
And also how to get to the console.

File: AWS.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
+# AWS Deployment
```
Comment (dlabordus) [inline AWS.md:1]: The file is double now :-)

Maybe add something to the README.md of the root pointing to `aws/README.md`

Comment (pascalwilbrink) [reply AWS.md]: Fixed

File: AWS.md (hunk 1 above)
Comment (pascalwilbrink) [reply AWS.md]: I tried to document this, but the AWS console is undergoing changes (for the second time).
I tried to document it more functional

**Pull request closed without merging**